girder/cumulus/server/models/aws.py

```diff
@@ -132,12 +133,14 @@ def validate(self, doc):
 
         return doc
 
-    def create_profile(self, userId, name, access_key_id, secret_access_key,
-                       region_name, availability_zone, public_ips):
+    def create_profile(self, userId, name, profile_type, access_key_id,
+                       secret_access_key, region_name, availability_zone,
+                       public_ips):
 
         user = getCurrentUser()
         profile = {
             'name': name,
+            'type': profile_type,
             'accessKeyId': access_key_id,
```

Comment:
This is a look towards more generic profiles.  Currently ```type``` is what determines the Provider class we load. What did you have in mind for naming?